README.md

```diff
@@ -85,7 +85,8 @@ $ vi ycsb.cpp
 For running the indexes on Intel Optane DC Persistent Memory, we will use 
 [libvmmalloc](http://pmem.io/pmdk/manpages/linux/v1.3/libvmmalloc.3.html) to 
 transparently converts all dynamic memory allocations into Persistent Memory 
-allocations, mapped by pmem.
+allocations, mapped by pmem. For details regarding the PMDK implementation please
```

Comment:
Right now the PMDK implementation seems like a small afterthought. You want a seperate para/point for it so that it is highlighted -- it is a big change!